### PR TITLE
dev-tunnel changes

### PR DESCRIPTION
**Pull request by ChetanSharma-msft**

### CODE REVIEW

File: samples/account-linking/csharp/README.md [hunk 1]
```diff
@@ -37,13 +37,20 @@ There are example in-memory and Azure implementations of the token persistance.
   ```bash
   dotnet --version
   ```
-- [ngrok](https://ngrok.com/download) (For local environment testing) Latest (any other tunneling software can also be used)
+
+- [devtunnel](https://aka.ms/TunnelsCliDownload/win-x64) or [ngrok](https://ngrok.com/download) (For local environment testing) latest version version (any other tunneling software can also be used).
```
Comment (curib) [inline samples/account-linking/csharp/README.md:41]: Should the aka link point to this link instead: https://learn.microsoft.com/azure/developer/dev-tunnels/get-started?tabs=windows

The current link automatically downloads the Windows executable. The ngrok link shows the different options to get ngrok based on your operating system.

File: samples/account-linking/csharp/README.md [hunk 2]
```diff
@@ -37,13 +37,20 @@ There are example in-memory and Azure implementations of the token persistance.
...
   
   run ngrok locally
   ```bash
   ngrok http 5001 --host-header="localhost:5001"
   ```
 
+  Alternatively, you can also use the `dev tunnel`. Please follow [Create and host a Dev tunnel](https://learn.microsoft.com/en-us/azure/developer/dev-tunnels/get-started?tabs=windows) and host the tunnel with anonymous user access command as shown below:
```
Comment (curib) [inline samples/account-linking/csharp/README.md:48]: Alternatively, you can also use dev tunnels. Please follow [Create and host a dev tunnel](https://learn.microsoft.com/en-us/azure/developer/dev-tunnels/get-started?tabs=windows) and host the tunnel with anonymous user access command as shown below:

File: samples/app-anonymous-users/csharp/Readme.md [hunk 3]
```diff
@@ -129,7 +135,7 @@ This sample shows anonymous users support in meeting apps.
    - `{{Microsoft-App-id}}` - Generated from Step 1 (Application (client) ID)is the application app id
    - `{{TenantId}}` - Generated from Step 1(Directory (tenant) ID) is the tenant id
    - `{{MicrosoftAppPassword}}` - Generated from Step 1.14, also referred to as Client secret
-   - `{{domain-name}}` - Your application's base url. E.g. https://12345.ngrok-free.app if you are using ngrok.
+   - `{{domain-name}}` - Your application's base url. E.g. https://12345.ngrok-free.app if you are using ngrok and if you are using dev tunnel, your URL will be like: https://12345.devtunnels.ms.
```
Comment (curib) [inline samples/app-anonymous-users/csharp/Readme.md:138]: dev tunnels instead of dev tunnel

**Review by curib — COMMENTED** [review]
These readmes look great! I added in some suggestions to the first readme. Looks like the same suggestions will apply to the other readmes. Thank you!

File: samples/account-linking/csharp/README.md (hunk 1 above)
Comment (Jegadeesh-MSFT) [reply samples/account-linking/csharp/README.md]: Fixed

File: samples/account-linking/csharp/README.md (hunk 2 above)
Comment (Jegadeesh-MSFT) [reply samples/account-linking/csharp/README.md]: Fixed as per request.

File: samples/app-anonymous-users/csharp/Readme.md (hunk 3 above)
Comment (Jegadeesh-MSFT) [reply samples/app-anonymous-users/csharp/Readme.md]: Fixed as per request.

**Review by curib — APPROVED** [review]

**Review by Harikrishnan-MSFT — APPROVED** [review]

**Review by Mohammed-MSFT — APPROVED** [review]
Changes looks fine approving it.

**Review by Prithvi-MSFT — APPROVED** [review]

**Pull request merged**